src/main/docker/artemis/Dockerfile

```
@@ -0,0 +1,56 @@
+#-----------------------------------------------------------------------------------------------------------------------
+# build stage
+#-----------------------------------------------------------------------------------------------------------------------
+FROM eclipse-temurin:17-jdk as builder
+
+WORKDIR /opt/artemis
+# so far just using the .dockerignore to define what isn't necessary here
+COPY . .
+
+RUN ./gradlew -i --stacktrace --no-daemon -Pprod -Pwar clean bootWar
+
+#-----------------------------------------------------------------------------------------------------------------------
+# runtime stage
+#-----------------------------------------------------------------------------------------------------------------------
+FROM eclipse-temurin:17-jre as runtime
```

```suggestion
FROM docker.io/library/eclipse-temurin:17-jdk as runtime
```
We actually changed back to the JDK as base image, since plagiarism checks for Java won’t work without.

---

Also changed the other `FROM` and `images` fields to explicitly use the registry `docker.io`.